common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -70,7 +70,7 @@ export type WarningInhibitor = (warning: RollupWarning) => boolean;
 
 function matchesPathSegments(str: string | undefined, segments: string[]): boolean {
   // Reported warnings use "/"
```

Comment:
```suggestion
```

This comment seems to indicate that we shouldn't expect `\` as a separator here, but there's no downside to accepting it I think.